appveyor.txt

```diff
@@ -1,6 +1,6 @@
 clone_script:
 - cmd: >-
-    git clone -q --branch=master https://github.com/alphagov/notifications-net-client.git C:\projects\notifications-net-client
+    git clone -q --branch=migrate-to-dotnet-core https://github.com/alphagov/notifications-net-client.git C:\projects\notifications-net-client
```

Comment:
Yes, it's because appveyor is not picking up the yml file when I created it, it would otherwise only pick up the static appveyor config on master. So will need to create another PR to point it back at master before deleting the branch.